Weather App/UI/SettingsVC/SettingsCoordinator.swift

```diff
@@ -20,9 +22,57 @@ class SettingsCoordinator: Coordinator {
         let viewController = SettingsViewController()
         let viewModel = SettingsViewModel()
         
-        //TODO: setup viewModel
+        viewModel.onEnter = { [weak self] in
+            if let loadedData = self?.loadSettingsData() {
+                viewController.settingsView.setupData(data: loadedData)
```

Comment:
I see, here, the coordinator is calling methods on the ViewController's view? Is that the way to go?

Reply:
Nope, the settingsView is now updated through the viewController